underfs/swift/src/main/java/alluxio/underfs/swift/SwiftUnderFileSystem.java

```diff
@@ -159,6 +165,26 @@ public SwiftUnderFileSystem(AlluxioURI uri) {
       container.create();
     }
     mContainerPrefix = Constants.HEADER_SWIFT + mContainerName + PATH_SEPARATOR;
+
+    // Assume the Swift user name has 1-1 mapping to Alluxio username.
+    mAccountOwner = Configuration.get(PropertyKey.SWIFT_USER_KEY);
+    short mode = (short) 0;
+    // If there is any container ACL for the Swift user, translates it to Alluxio permission.
+    if (container.getContainerReadPermission().contains(mAccountOwner)
```

Comment:
also, what happens when read_acl is such that read permission is granted to all users with a given tenant name. the acl syntax seems to have changed from v1 to v2. refer parse_acl in https://github.com/openstack/swift/blob/master/swift/common/middleware/acl.py